integration_tests/tests/apply/risks_and_needs/health-needs/physical_health.cy.ts

```diff
@@ -60,12 +61,20 @@ context('Visit "Physical health" page', () => {
     Page.verifyOnPage(PhysicalHealthPage, this.application)
   })
 
-  //  Scenario: navigate to next page in health needs task
-  //    When I continue to the next task / page
+  //  Scenario: complete page and navigate to next page in health needs task
+  //    When I complete the physical health page questions
+  //    And I continue to the next task / page
   //    Then I see the "mental health" page
   it('navigates to the next page (mental health)', function test() {
     PhysicalHealthPage.visit(this.application)
     const page = new PhysicalHealthPage(this.application)
+
+    page.describePhysicalNeeds()
+    page.describeTreatment()
+    page.describeMedication()
+    page.provideIndependentLivingInfo()
+    page.describeAdditionalSupportNeeded()
```

Comment:
I've described the 5 questions which are being answered here as I think it's important to be clear (here in the scenario) about the information which is being communicated on this 'page'. This is the "behaviour" at an appropriate level of detail, I think. As an applicant I'm answering the following 5 questions: about physical needs, treatment, medication etc. The implementation details -- what quantity and type of form fields are used -- is not relevant, as is handled in the step helper.

Reply:
In the E2E tests -- by contrast -- we are satisfied with a single line: something like `completePhysicalHeathPage()`